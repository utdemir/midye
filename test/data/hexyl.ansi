┌────────┬─────────────────────────┬─────────────────────────┬────────┬────────┐
│[38;5;242m00000000[0m│ [36m61 [0m[32m0a [0m[35m1b [0m[36m5b [0m[36m30 [0m[36m6d [0m[35m1b [0m[36m5b [0m┊ [36m30 [0m[36m31 [0m[36m3b [0m[36m33 [0m[36m34 [0m[36m6d [0m[36m62 [0m[35m1b [0m│[36ma[0m[32m_[0m[35m•[0m[36m[[0m[36m0[0m[36mm[0m[35m•[0m[36m[[0m┊[36m0[0m[36m1[0m[36m;[0m[36m3[0m[36m4[0m[36mm[0m[36mb[0m[35m•[0m│
│[38;5;242m00000010[0m│ [36m5b [0m[36m30 [0m[36m6d [0m[32m0a [0m[36m63 [0m[32m0a [0m[35m1b [0m[36m5b [0m┊ [36m34 [0m[36m30 [0m[36m3b [0m[36m33 [0m[36m31 [0m[36m3b [0m[36m30 [0m[36m31 [0m│[36m[[0m[36m0[0m[36mm[0m[32m_[0m[36mc[0m[32m_[0m[35m•[0m[36m[[0m┊[36m4[0m[36m0[0m[36m;[0m[36m3[0m[36m1[0m[36m;[0m[36m0[0m[36m1[0m│
│[38;5;242m00000020[0m│ [36m6d [0m[36m66 [0m[36m6f [0m[36m6f [0m[36m62 [0m[36m61 [0m[36m72 [0m[35m1b [0m┊ [36m5b [0m[36m30 [0m[36m6d [0m[32m0a [0m            │[36mm[0m[36mf[0m[36mo[0m[36mo[0m[36mb[0m[36ma[0m[36mr[0m[35m•[0m┊[36m[[0m[36m0[0m[36mm[0m[32m_[0m    │
└────────┴─────────────────────────┴─────────────────────────┴────────┴────────┘
