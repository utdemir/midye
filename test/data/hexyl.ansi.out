┌────────┬─────────────────────────┬─────────────────────────┬────────┬────────┐
TS_LF
│
TS_SGR [38,5,242]
00000000
TS_SGR [0]
│ 
TS_SGR [36]
61 
TS_SGR [0]
TS_SGR [32]
0a 
TS_SGR [0]
TS_SGR [35]
1b 
TS_SGR [0]
TS_SGR [36]
5b 
TS_SGR [0]
TS_SGR [36]
30 
TS_SGR [0]
TS_SGR [36]
6d 
TS_SGR [0]
TS_SGR [35]
1b 
TS_SGR [0]
TS_SGR [36]
5b 
TS_SGR [0]
┊ 
TS_SGR [36]
30 
TS_SGR [0]
TS_SGR [36]
31 
TS_SGR [0]
TS_SGR [36]
3b 
TS_SGR [0]
TS_SGR [36]
33 
TS_SGR [0]
TS_SGR [36]
34 
TS_SGR [0]
TS_SGR [36]
6d 
TS_SGR [0]
TS_SGR [36]
62 
TS_SGR [0]
TS_SGR [35]
1b 
TS_SGR [0]
│
TS_SGR [36]
a
TS_SGR [0]
TS_SGR [32]
_
TS_SGR [0]
TS_SGR [35]
•
TS_SGR [0]
TS_SGR [36]
[
TS_SGR [0]
TS_SGR [36]
0
TS_SGR [0]
TS_SGR [36]
m
TS_SGR [0]
TS_SGR [35]
•
TS_SGR [0]
TS_SGR [36]
[
TS_SGR [0]
┊
TS_SGR [36]
0
TS_SGR [0]
TS_SGR [36]
1
TS_SGR [0]
TS_SGR [36]
;
TS_SGR [0]
TS_SGR [36]
3
TS_SGR [0]
TS_SGR [36]
4
TS_SGR [0]
TS_SGR [36]
m
TS_SGR [0]
TS_SGR [36]
b
TS_SGR [0]
TS_SGR [35]
•
TS_SGR [0]
│
TS_LF
│
TS_SGR [38,5,242]
00000010
TS_SGR [0]
│ 
TS_SGR [36]
5b 
TS_SGR [0]
TS_SGR [36]
30 
TS_SGR [0]
TS_SGR [36]
6d 
TS_SGR [0]
TS_SGR [32]
0a 
TS_SGR [0]
TS_SGR [36]
63 
TS_SGR [0]
TS_SGR [32]
0a 
TS_SGR [0]
TS_SGR [35]
1b 
TS_SGR [0]
TS_SGR [36]
5b 
TS_SGR [0]
┊ 
TS_SGR [36]
34 
TS_SGR [0]
TS_SGR [36]
30 
TS_SGR [0]
TS_SGR [36]
3b 
TS_SGR [0]
TS_SGR [36]
33 
TS_SGR [0]
TS_SGR [36]
31 
TS_SGR [0]
TS_SGR [36]
3b 
TS_SGR [0]
TS_SGR [36]
30 
TS_SGR [0]
TS_SGR [36]
31 
TS_SGR [0]
│
TS_SGR [36]
[
TS_SGR [0]
TS_SGR [36]
0
TS_SGR [0]
TS_SGR [36]
m
TS_SGR [0]
TS_SGR [32]
_
TS_SGR [0]
TS_SGR [36]
c
TS_SGR [0]
TS_SGR [32]
_
TS_SGR [0]
TS_SGR [35]
•
TS_SGR [0]
TS_SGR [36]
[
TS_SGR [0]
┊
TS_SGR [36]
4
TS_SGR [0]
TS_SGR [36]
0
TS_SGR [0]
TS_SGR [36]
;
TS_SGR [0]
TS_SGR [36]
3
TS_SGR [0]
TS_SGR [36]
1
TS_SGR [0]
TS_SGR [36]
;
TS_SGR [0]
TS_SGR [36]
0
TS_SGR [0]
TS_SGR [36]
1
TS_SGR [0]
│
TS_LF
│
TS_SGR [38,5,242]
00000020
TS_SGR [0]
│ 
TS_SGR [36]
6d 
TS_SGR [0]
TS_SGR [36]
66 
TS_SGR [0]
TS_SGR [36]
6f 
TS_SGR [0]
TS_SGR [36]
6f 
TS_SGR [0]
TS_SGR [36]
62 
TS_SGR [0]
TS_SGR [36]
61 
TS_SGR [0]
TS_SGR [36]
72 
TS_SGR [0]
TS_SGR [35]
1b 
TS_SGR [0]
┊ 
TS_SGR [36]
5b 
TS_SGR [0]
TS_SGR [36]
30 
TS_SGR [0]
TS_SGR [36]
6d 
TS_SGR [0]
TS_SGR [32]
0a 
TS_SGR [0]
            │
TS_SGR [36]
m
TS_SGR [0]
TS_SGR [36]
f
TS_SGR [0]
TS_SGR [36]
o
TS_SGR [0]
TS_SGR [36]
o
TS_SGR [0]
TS_SGR [36]
b
TS_SGR [0]
TS_SGR [36]
a
TS_SGR [0]
TS_SGR [36]
r
TS_SGR [0]
TS_SGR [35]
•
TS_SGR [0]
┊
TS_SGR [36]
[
TS_SGR [0]
TS_SGR [36]
0
TS_SGR [0]
TS_SGR [36]
m
TS_SGR [0]
TS_SGR [32]
_
TS_SGR [0]
    │
TS_LF
└────────┴─────────────────────────┴─────────────────────────┴────────┴────────┘
TS_LF

